src/index.html

```diff
@@ -1,13 +1,401 @@
+                >
+              </div>
+              <h3 class="testimonial__name">
+                <a href="#" class="link">
+                  Max Henry
+                </a>
+              </h3>
+              <div class="testimonial__role">
+                Chief Design Officer
+              </div>
+              <div class="testimonial__about">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </div>
+            </div>
+
+
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__label">
+                  Phone
+                </div>
+                <div class="contact-us__link">
+                  <a href="tel: +1 234 5555-55-55" class="link link--input">+1 234 5555-55-55</a>
+                </div>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__label">
+                  Email
+                </div>
+                <div class="contact-us__link">
+                  <a href="mailto: hello@miami.com" class="link link--input">
+                    hello@miami.com
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info">
+                <div class="contact-us__label">
+                  Address
+                </div>
+                <div class="contact-us__link">
+                  <a href="mailto: hello@miami.com" class="link link--input">
+                    400 first ave. n. <br>
+                    suite 700 <br>
+                    Minneapolis, MN 55401
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="index.html" method="post">
+                <input
+                  name="name"
+                  type="text"
+                  class="form-field contact-us__field"
+                  placeholder="Name"
+                >
+                <input
+                  name="email"
+                  type="text"
+                  class="form-field contact-us__field"
+                  placeholder="Email"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contact-us__field
+                    form-field
+                    form-field--textarea"
+                ></textarea>
+                <button type="submit" class="button button--primary">Consultation</button>
```

Comment:
Checklist 15. Page shouldn't be reloaded on form submit [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)